Makefile

```diff
@@ -1,53 +1,36 @@
 # Copyright (c) The Jaeger Authors.
 # SPDX-License-Identifier: Apache-2.0
 #
-# cSpell:ignore docsy htmltest refcache
+# cSpell:ignore refcache
 
-DEPLOY_PRIME_URL ?= http://localhost
-HUGO_THEME   = jaeger-docs
-THEME_DIR    := themes/$(HUGO_THEME)
 HTMLTEST     ?= htmltest
 HTMLTEST_DIR ?= tmp/.htmltest
 
+# Use $(HTMLTEST) in PATH, if available; otherwise, we'll get a copy
+ifeq (, $(shell which $(HTMLTEST)))
```

Comment:
Doesn't this make the build non-repeatable?

Reply:
It is a convenience for developers who might have htmltest installed differently.
The build remains repeatable in our CI and Netlify environments (since they won't have htmltest installed by default). IMHO, that is what is important.

If you have something else in mind, let me know.